WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -51,11 +51,27 @@
     // the post they're attached to, allowing us to update the post with the media URLs in a single batch at the end
     private static final ConcurrentHashMap<Integer, List<MediaModel>> sCompletedMediaByPost = new ConcurrentHashMap<>();
 
+    // This will hold a map of postID and the corresponding last error if any error occured while uploading
+    private static final ConcurrentHashMap<Integer, UploadError> sFailedUploadPosts = new ConcurrentHashMap<>();
```

Comment:
Want to add a `TODO` here like [this one](https://github.com/wordpress-mobile/WordPress-Android/pull/6343/files#diff-fd1074f533e624627941f15b88dd3277R38) about moving this eventually to a `PostUploadTable`? And also list it in https://github.com/wordpress-mobile/WordPress-Android/issues/6352?